src/main/java/io/cdap/plugin/salesforce/SalesforceTransformUtil.java

```diff
@@ -43,7 +44,7 @@ public static Object transformLogicalType(String fieldName, Schema.LogicalType l
       case TIMESTAMP_MICROS:
         return TimeUnit.MILLISECONDS.toMicros(Instant.parse(value).toEpochMilli());
       case TIME_MICROS:
-        return TimeUnit.NANOSECONDS.toMicros(LocalTime.parse(value).toNanoOfDay());
+        return TimeUnit.NANOSECONDS.toMicros(LocalTime.parse(value, DateTimeFormatter.ISO_TIME).toNanoOfDay());
```

Comment:
Is this util used for both source and sink?

Reply:
No, this is just for source. Does the JIRA ticket also include Sink plugin as well?

Reply:
so `SalesforceTransformUtil.java` is not used in batch sink? Is that a correct statement?

Reply:
SalesforceTransformUtil.java is not used in batch sink.